_docs-v5/view-api/view-render-hooks.md

```diff
@@ -7,9 +7,9 @@ title: View Render Hooks
 
 `viewClassNames` - a [ClassName Input](classname-input) for adding classNames to the root view element. called whenever the view changes.
 
-`viewDidMount` - called right after the view has been added to the DOM
+`viewDidMount` - called right after the view has been added to the DOM or in specific [view changes](#view-changes).
```

Comment:
saying "**during** specific view changes" would be clearer.

what what does "specific" mean here.

it's better phrased as:

> Called right after the view has been added to the DOM. This can happen when the calendar first loads, or when the user switches to a new view.